app/src/main/AndroidManifest.xml

```diff
@@ -34,6 +28,16 @@
         <activity
             android:name=".view.SettingsActivity"
             android:label="@string/settings" />
-    </application>
+        <activity
+            android:name=".view.InitActivity"
```

Comment:
I think we still want to specify `android:screenOrientation="portrait"` for the splash screen activity.

Currently when I rotate the device during the initial activity the animation restarts and upon completing the app subsequently crashes.